src/components/resume/workResumeEntry.svelte

```diff
@@ -1,38 +1,38 @@
 <script lang="ts">
   import type { Highlight } from '@src/data/data';
-  import { derived, type Writable } from 'svelte/store';
   import { tagsStore } from '@src/data/tag';
-  import { isHighlightVisible, dateInputToDecoratedString } from '@src/util/resumeUtils';
+  import { isHighlightVisible } from '@src/util/resumeUtils';
 
-  export let name: Writable<string>;
-  export let position: Writable<string>;
-  export let startDate: Writable<string>;
-  export let endDate: Writable<string>;
-  export let highlights: Writable<Array<Highlight>>;
-
-  let pos = derived(position, ($position) => ($position ? $position : 'Position'));
-  let n = derived(name, ($name) => ($name ? $name : 'Name of employer'));
-  let sd = derived(startDate, ($startDate) => dateInputToDecoratedString($startDate, 'Start date'));
-  let ed = derived(endDate, ($endDate) => dateInputToDecoratedString($endDate, 'End date'));
+  export let i: number;
+  export let name: string;
+  export let position: string;
+  export let startDate: string;
+  export let endDate: string;
+  export let highlights: Highlight[];
```

Comment:
At first, I thought having these components have multiple inputs like this was an annoying idea, so I opted to do something like this to simplify the assignments
```ts
export let w: Work; // imported work object
let name = w.name; // assign different vars to the properties of the work object
let position = w.position
// ...
```
However, this kept breaking my automatically updating fields. I soon remembered that Svelte's reactivity applies to *assigned variables*. So, if I assigned `name` to something else like `name = 'blah'`, then the name would change in the UI. If I updated the `workStore` that controlled those values, the variables I assigned would still remain the same.

Expecting specific properties to be passed into the component prevents me from making assignments that'll accidentally remove reactivity from a component, and keeps me from appending all of the variables I want to manipulate with the work object, such as `w.name`, `w.position` and so on. I should remember to stick with this pattern from now on.